task/host-sp-comms/src/tx_buf.rs

```diff
@@ -139,10 +139,16 @@ impl TxBuf {
     }
 
     /// Should we be sending periodic 0 bytes?
+    #[cfg(feature = "gimlet")]
     pub(crate) fn should_send_periodic_zero_bytes(&self) -> bool {
         matches!(self.state, State::Idle)
     }
 
+    #[cfg(not(feature = "gimlet"))]
+    pub(crate) fn should_send_periodic_zero_bytes(&self) -> bool {
```

Comment:
Nit: maybe `feature = "grapefruit"` instead, so that we notice when we add a new target?

(e.g. after adding Cosmo support, it will become `any(feature = "grapefruit", feature = "cosmo")`)